pkg/resources/schema.go

```diff
@@ -29,268 +33,380 @@ var schemaSchema = map[string]*schema.Schema{
 		Description: "The database in which to create the schema.",
 		ForceNew:    true,
```

Comment:
Imo there should be some diff suppression on `name` and `database` in case we get strange quoting from Snowflake.